articles/container-registry/container-registry-oci-artifacts.md

```diff
@@ -51,6 +41,52 @@ az acr login --name myregistry
 > [!NOTE]
 > `az acr login` uses the Docker client to set an Azure Active Directory token in the `docker.config` file. The Docker client must be installed and running to complete the individual authentication flow.
 
+### Sign in with ORAS
+
+This section shows options to sign into the registry. Choose one method below appropriate for your environment.
+
+Run  `oras login` to authenticate with the registry. You may pass [registry credentials](container-registry-authentication.md) appropriate for your scenario, such as service principal credentials, user identity, or a repository-scoped token (preview).
+
+- Authenticate with your [individual Azure AD identity](container-registry-authentication.md?tabs=azure-cli#individual-login-with-azure-ad) to use an AD token.
+
+  ```azurecli
+  USER_NAME="00000000-0000-0000-0000-000000000000"
+  PASSWORD=$(az acr login --name $ACR_NAME --expose-token --output tsv --query accessToken)
+  ```
+
+- Authenticate with a [repository scoped token](container-registry-repository-scoped-permissions.md) (Preview) to use non-AD based tokens.
```

Comment:
I appreciate the extra text, but I wonder if this adds complexity. Do users already know which auth pattern to use? Should that be a section for choosing which auth pattern?

> - Authenticate with a [repository scoped token](container-registry-repository-scoped-permissions.md) (Preview) ~to use non-AD based tokens.~

Reply:
I just copied these three authentication patterns from another [guide](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-oras-artifacts#sign-in-with-oras). Now I think it would be better to keep only one default pattern as users might be confused to select from three. Another option is to provide three authentication patterns in different tabs. 

This is not a blocking problem. Maybe we can discuss it offline and adjust the format in another PR.